### DISCUSSION TITLE
alist的webdav功能是否不支持流式传输？

### DISCUSSION THREAD

**Discussion by goldsoft8888** [AlistGo/alist#5368] [General]
alist的webdav功能是否不支持流式传输？
我用rclone 挂载alist的webdav，会提示如下：
2023/10/12 06:18:34 NOTICE: webdav root '网盘': --vfs-cache-mode writes or full is recommended for this remote as it can't stream

--vfs-cache-mode writes 加入这个参数是正常的，但会占用大量的CPU、内存、磁盘，所以我想采用流式传输，但我不确定这是rclone的问题，还是alist的问题。 alist 挂载云盘的流式上传非常棒，我想让rclone也实现类似的功能，不在磁盘上缓存。

**Comment by goldsoft8888** [comment]
如果可以不用rclone，只用alist实现映射smb或本地文件夹，会更好，愿意为此捐赠

**Reply by rakaki** [reply]
可是直接在支持webdav的播放器里添加alist是可以流式传输的
